docs/en/04_Changelogs/4.0.0.md

```diff
@@ -892,6 +892,11 @@ A very small number of methods were chosen for deprecation, and will be removed
   * Any extension declared via `versionableExtensions` config on Versioned dataobject must now 
     `VersionableExtension` interface at a minimum. `Translatable` has been removed from default
     `versionableExtensions`
+  * The default CMS delete behaviour for versioned dataobjects is now to delete from both draft
+    and live stages, and to save to the archive. There is no longer a separate "Archive" action.
+  * Any writes to versioned dataobjects (in either Draft or Live) will always write to the draft
+    (main) table as a source of truth. This means that records created in Live mode will always be
+    available to the CMS and in draft mode.
```

Comment:
Should this read `...will always be available to the CMS in draft mode.` (without the `and`)?